packages/x-components/tests/e2e/cucumber/next-queries/next-queries.spec.ts

```diff
@@ -18,14 +20,22 @@ Given(
         }
       }
     };
-    cy.visit('/test/next-queries', {
-      qs: {
-        xModules: JSON.stringify(config)
-      }
-    });
   }
 );
 
+And('next queries response being mock {string}', (mockName: string) => {
```

Comment:
Change the text with something like: `next queries API should respond with {string}`